### ISSUE TITLE
NTR: Ultrasound: duplicate uterine cervix  (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8921]
New term request Ultrasound: duplicate uterine cervix  (elthat_100730014923): Two cervices on uterus

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000414
- label: Duplicate uterine cervix
- parents: [Abnormal uterine cervix morphology (HP:0012888)](https://hpo.jax.org/app/browse/term/HP:0012888)
- pmid: PMID:30644250
- def: A congenital developmental defect characterized by the presence of two cervices on the uterus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8921
- synonyms: Cervical duplication(NONE)
